Lombiq.Tests.UI/Extensions/ReliabilityUITestContextExtensions.cs

```diff
@@ -234,10 +236,19 @@ public static class ReliabilityUITestContextExtensions
         TimeSpan? timeout = null,
         TimeSpan? interval = null)
     {
-        var lastScrollPosition = context.GetScrollPosition();
+        Point? lastScrollPosition = null;
+
+        Thread.Sleep(interval ?? RetrySettings.Interval);
```

Comment:
Why is this needed in the beginning? The practice for all such retried operations is that you check first, and if you don't succeed, then wait and then retry.

Reply:
Adding a sleep in the beginning is arbitrary though. It might very well be finished by reaching this point.

Reply:
This caused problems on GH windows-latest, but disabling smooth scrolling (--disable-smooth-scrolling) in the chrome's command line solved the issue, and finally, I removed this delay.

Reply:
That's a much better solution.